extensions/packageManager/src/node/index.ts

```diff
@@ -38,11 +43,85 @@ const normalizeFeed = (feed) => {
         source: 'nuget',
       };
     });
+  } else if (feed.resources) {
+    // this is actually a myget feed that points to the feed we want...
+    const queryEndpoint = feed.resources.find((resource) => resource['@type'] === 'SearchQueryService');
+    if (queryEndpoint) {
+      const raw = await axios.get(queryEndpoint['@id']);
+      return normalizeFeed(raw.data);
+    } else {
+      return [];
+    }
   } else {
+    console.error('Unknown feed format!', feed);
     return null;
   }
 };
 
+
+const getPackageInfo = async (name, url) => {
+
+  // available versions should be folders underneath this folder
+  let versions;
+  try {
+    versions = readdirSync(url, {withFileTypes: true}).filter(f=>f.isDirectory()).map(f=>f.name);
+    if (versions.length === 0) {
+      throw new Error('version list is empty');
+    }
+    versions = semverSort.desc(versions);
+  } catch(err) {
+    throw new Error(`Could not find versions of local package ${ name } at ${ url }`)
+  }
+
+  // can read from the nuspec file in the latest to get other info
+  let metadata = {};
+  try {
+    const pathToNuspec = path.join(url, versions[0], `${ name }.nuspec`);
+
+    const xml = readFileSync(pathToNuspec,'utf8');
+
+    const parsed = await parseStringPromise(xml);
+    metadata = {
+      id: parsed.package.metadata[0].id[0],
+      version: parsed.package.metadata[0].version[0],
+      authors: parsed.package.metadata[0].authors,
+      projectUrl: parsed.package.metadata[0].projectUrl[0],
+      description: parsed.package.metadata[0].description[0],
+      tags: parsed.package.metadata[0].tags[0].split(/\s/),
+      versions: versions.map(v=> {return {version: v}}),
+      source: 'local',
+      language: 'c#',
```

Comment:
What's this for? If it is going to be shown to users, in general in the dotnet world we use `C#` with uppercase.

Reply:
This is used to determine if the package uses the same programming language as the runtime. It is not displayed to the user.